packages/input-edit/src/components/InputEditTable/InputEditTable.js

```diff
@@ -104,15 +146,37 @@ const DataCell = ({nodes, column}) => {
   </td>
 }
 
-const InputCell = ({index, nodes, column, updateValue}) => {
+const loadValue = (nodes, column) => {
+  const pathElements = column.children[0].path.split('.')
+  return resolvePath(nodes, pathElements)
+}
+
+const resolvePath = (current, pathElements) => {
```

Comment:
i would move this method in some kind of util file and write unit a unit tests. that will also help to kind of document what is happening here